ArchiSteamFarm/Storage/GlobalConfig.cs

```diff
@@ -422,7 +424,7 @@ public sealed class GlobalConfig {
 	public bool ShouldSerializeIPCPasswordFormat() => !Saving || (IPCPasswordFormat != DefaultIPCPasswordFormat);
 
 	[UsedImplicitly]
-	public bool ShouldSerializeLicenseID() => !Saving || ((LicenseID != DefaultLicenseID) && (LicenseID != Guid.Empty));
```

Comment:
You need similar logic to `IsIPCPasswordSet`/`IsWebProxyPasswordSet` for achieving that, otherwise you won't properly allow user to unset the LicenseID - https://github.com/JustArchiNET/ArchiSteamFarm/blob/main/ArchiSteamFarm/IPC/Controllers/Api/ASFController.cs#L120

Reply:
@Abrynos you're still missing logic mentioned in https://github.com/JustArchiNET/ArchiSteamFarm/blob/main/ArchiSteamFarm/IPC/Controllers/Api/ASFController.cs#L120 but I can handle it myself I guess :slightly_smiling_face:

Reply:
Ah shit I totally forgot about this one. I'm sorry.